packages/tour-kit/package.json

```diff
@@ -43,8 +43,6 @@
 		"@wordpress/primitives": "^3.5.0",
 		"@wordpress/react-i18n": "^3.0.4",
 		"classnames": "^2.3.1",
-		"react": "^17.0.2",
-		"react-dom": "^17.0.2",
```

Comment:
Why would we have these defined as deps in first place if we use `@wordpress/dom` and `@wordpress/element` wrappers for React/React-dom?

Reply:
Hmm, they were added in https://github.com/Automattic/wp-calypso/commit/666e1bf13f78b4304de41d7a5a666d6ec8d9d6df and https://github.com/Automattic/wp-calypso/commit/caeb4acf8574a93ed02aacb9c42b8494f6777879.

@chriskmnds @escapemanuele 👋  Any specific reasons for adding React/React-dom?

Reply:
Hi @chihsuan, I see `react` being used only for small things, like for `MutableRefObject` [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/tour-kit/src/components/keyboard-navigation.tsx#L17), `FunctionComponent` [here](https://github.com/Automattic/wp-calypso/blob/add/react-router-to-hc/packages/tour-kit/src/components/tour-kit.tsx#L16), some code [here](https://github.com/Automattic/wp-calypso/blob/add/react-router-to-hc/packages/tour-kit/src/types.ts#L38) and for `ErrorInfo` [here](https://github.com/Automattic/wp-calypso/blob/add/react-router-to-hc/packages/tour-kit/src/error-boundary.tsx#L1).
So, I think those can be changed and next the dependencies removed!

Reply:
@escapemanuele Thank you for the quick reply! 

Looks like they're only used for type annotations. 

I believe we could keep them. Even if we remove react & react-dom from dependencies, we could still resolve them properly because [root package.json includes the required react dependencies](https://github.com/Automattic/wp-calypso/blob/trunk/package.json#L157).

Reply:
Hi @simison Could you review it again when you get a chance please?